jmxtrans-output/jmxtrans-output-log4j/pom.xml

```diff
@@ -47,10 +47,12 @@
 		<dependency>
 			<groupId>ch.qos.logback</groupId>
 			<artifactId>logback-classic</artifactId>
+			<optional>true</optional>
 		</dependency>
 		<dependency>
 			<groupId>ch.qos.logback</groupId>
 			<artifactId>logback-core</artifactId>
+			<optional>true</optional>
```

Comment:
This does not seem to be an optional dependency. At least `TimeBasedRollingKeyOutWriter` depends on it.